src/solvers/dgsem_tree/dg_3d_parabolic.jl

```diff
@@ -338,10 +339,11 @@ function prolong2boundaries!(cache_parabolic, flux_viscous,
     return nothing
 end
 
-function calc_viscous_fluxes!(flux_viscous, gradients, u_transformed,
+function calc_viscous_fluxes!(flux_viscous::Vector{Array{uEltype, 5}},
+                              gradients::Vector{Array{uEltype, 5}}, u_transformed,
```

Comment:
See above on `flux_viscous`, same comments on `gradients`

Reply:
At this place not, this is some left-over.